### PR TITLE
changed from jpg to webp

### PR DESCRIPTION
**Pull request by yaroslavbilokin**

### CODE REVIEW

File: client/cypress/integration/pdp.spec.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...mnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur.","propetries":[{"size":["S"],"_id":"5e8da1fcd9a7af033c9b1bcf","available":6,"sku":"TXWOEP11"},{"size":["L"],"_id":"5e8da1fcd9a7af033c9b1bd0","available":6,"sku":"PMO7DVZS"},{"size":["XL"],"_id":"5e8da1fcd9a7af033c9b1bd1","available":2,"sku":"S5J85D03"}],"price":144,"mrsp":207,"rate":3.86,"brand":"zori","catalog":"women","category":"dresses","color":"blue"}]'
```
Comment (oleksandrhodzevych) [inline client/cypress/integration/pdp.spec.js]: move to fixture

File: client/src/components/materials-page/Materials-page.js [hunk 2]
```diff
@@ -1,120 +1,120 @@
...
 
     useEffect(() => {
-        window.scrollTo(0,0)
-      }, []);
+        window.scrollTo(0, 0)
+    }, []);
 
-    return(
+    return (
         <div className='materials-box'>
```
Comment (oleksandrhodzevych) [inline client/src/components/materials-page/Materials-page.js]: move static html to new file

File: client/src/components/wish-list/Wish-list.js [hunk 3]
```diff
@@ -25,56 +25,56 @@ const Wishlist = ({removeFromWishlist}) => {
...
         <span className="item-description">
-        <ReadMoreReact text={item.description}
-          min={60}
-          ideal={100}
-          max={10000}
-          readMoreText="..."/> 
-            
-        <span className='full-description-wishlist'>{item.description}</span>
+          <ReadMoreReact text={item.description}
+            min={60}
```
Comment (oleksandrhodzevych) [inline client/src/components/wish-list/Wish-list.js]: move hard coded data

File: client/src/components/wish-list/Wish-list.js [hunk 4]
```diff
@@ -25,56 +25,56 @@ const Wishlist = ({removeFromWishlist}) => {
...
-            className='cart-btns shop-now-btn'
-          ><span>Wish Now</span>
-          </Button>
-          </Link>
-          </div>
-        </div>}
+            <div className='empty-cart'><img src='/images/empty-basket.png' alt='Your cart is empty'></img><br />
+              <Link style={{ textDecoration: 'none' }} key='shop-now-btn' to={`/`}>
+                <Button
+                  variant="dark"
```
Comment (oleksandrhodzevych) [inline client/src/components/wish-list/Wish-list.js:78]: alt='Your cart is empty' - hard code

**Review by oleksandrhodzevych — CHANGES_REQUESTED** [review]

File: client/cypress/integration/pdp.spec.js (hunk 1 above)
Comment (yaroslavbilokin) [reply client/cypress/integration/pdp.spec.js]: moved to fixture

File: client/src/components/materials-page/Materials-page.js (hunk 2 above)
Comment (yaroslavbilokin) [reply client/src/components/materials-page/Materials-page.js]: moved static html to new file

File: client/src/components/wish-list/Wish-list.js (hunk 3 above)
Comment (yaroslavbilokin) [reply client/src/components/wish-list/Wish-list.js]: moved hard coded data

File: client/src/components/wish-list/Wish-list.js (hunk 4 above)
Comment (yaroslavbilokin) [reply client/src/components/wish-list/Wish-list.js:78]: fixed

**Pull request merged**